src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -199,21 +204,295 @@ const testMe: ShippingTestQuery$rawResponse["me"] = {
   },
 }
 
+const pushMock = jest.fn()
+let isCommittingMutation
+const relayEnv = createMockEnvironment()
+
+// This config extracted so it can be used with both enzyme (setupTestWrapper)
+// and react-testing-library (setupTestWrapperTL) tests.
+// We need to pass in our own relayEnv to both <MockBoot /> and
+// renderWithRelay so that downstream QueryRenderers can use the test
+// environment. Otherwise SystemContext will create a new [real] environment.
+const testWrapperConfig = {
+  Component: (props: any) => (
+    <MockBoot
+      relayEnvironment={relayEnv}
+      user={{
+        id: "example-user-id",
+        name: "Logged In",
+        email: "loggedin@example.com",
+      }}
+      context={{ user: { id: "example-user-id" } }}
+    >
+      <ShippingFragmentContainer
+        router={{ push: pushMock } as any}
+        order={props.order}
+        me={props.me}
+        // @ts-ignore
+        isCommittingMutation={isCommittingMutation}
+      />
+    </MockBoot>
+  ),
+  query: graphql`
+    query ShippingTestQuery @raw_response_type @relay_test_operation {
+      order: commerceOrder(id: "unused") {
+        ...Shipping_order
+      }
+      me {
+        ...Shipping_me
+      }
+    }
+  `,
+}
+
+describe("Shipping [@testing-library/react]", () => {
```

Comment:
Is the `[@testing-library/react]` string necessary?

If we're going to migrate this stuff, lets just migrate everything all at once and not mix testing libraries inside the same file. Mixing like this makes the the chaotic order app situation worse for little gain (since there's no real need to refactor tests or change approaches). 

We should either refactor it all, or stick with the existing model and then refactor everything later all at once.